apps/server/src/modules/oauth-provider/uc/oauth-provider.uc.ts

```diff
@@ -1,7 +1,13 @@
 import { Injectable } from '@nestjs/common';
 import { OauthProviderService } from '@shared/infra/oauth-provider/index';
+import { RedirectResponse } from '@shared/infra/oauth-provider/dto';
 
 @Injectable()
 export class OauthProviderUc {
 	constructor(private readonly oauthProviderService: OauthProviderService) {}
+
+	logoutFlow(challenge: string): Promise<RedirectResponse> {
```

Comment:
We should consider putting this method into its own UC for consistency with the others or rename this one

Reply:
done